src/main/java/frc/robot/InitSubsystems.java

```diff
@@ -76,6 +82,27 @@ public static Drive initDriveSubsystem() {
             new ModuleIOSim(DriveConfiguration.getInstance().BackLeft),
             new ModuleIOSim(DriveConfiguration.getInstance().BackRight));
 
+      case MAPLESIM:
+
+        // Sim robot, instantiate physics sim IO implementations
+        RobotContainer.driveSim =
+            new SwerveDriveSimulation(DriveTrainSimulationConfig.Default(), new Pose2d());
```

Comment:
Since you are crashing into invisible objects, I think we may just not be setting the correct dimensions / configuration. Rather than setting the default, let's now try to make it for our robot